src/elements/Button/Button.styl

```diff
@@ -1,8 +1,19 @@
 @import "~@/assets/tokens/tokens.styl"
-
 @import "~@/styles/colors.styl"
 @import "~@/styles/font-reset.styl"
 @import "~@/styles/sizes.styl"
+@import "~@/styles/control.styl"
+@import "~@/patterns/LicenseBadge/LicenseBadge.styl"
```

Comment:
Why import a style from `LicenseBadge` into `Button`?

Reply:
I meant to fiddle with the logo inside Button to make it look good in spite of all the changes. But As it was an added bonus to my contribution it can be removed.

Reply:
```suggestion

```